src/apprt/gtk/Window.zig

```diff
@@ -79,6 +83,7 @@ pub fn init(self: *Window, app: *App) !void {
         .notebook = undefined,
         .context_menu = undefined,
         .toast_overlay = undefined,
+        .wayland = undefined,
```

Comment:
I think this needs to be set to null, as Wayland.init can return an error and if it does then wayland won't be set and will stay undefined. And if configChange is called then the undefined memory will be accessed

Reply:
If we try and initialize a connection to Wayland and it fails, perhaps that should be a fatal error?

Reply:
> If we try and initialize a connection to Wayland and it fails, perhaps that should be a fatal error?

This would probably be the ideal approach.